src/App.js

```diff
@@ -1,24 +1,46 @@
 import './App.scss';
+import 'toastr/build/toastr.min.css'
+import React, { useState } from 'react'
 import 'bootstrap/dist/css/bootstrap.min.css'
 import { HashRouter as Router, Switch, Route } from "react-router-dom";
-import {Entry, Dashboard, Account} from './pages'
-import {CreateRecipe, RecipeFeed, SingleRecipe} from './pages/RecipePages'
-import {CreateMealPlan, MealPlanFeed, SingleMealPlan} from './pages/MealPlanPages'
+import { Entry, Dashboard, Account } from './pages'
+import { CreateRecipe, RecipeFeed, SingleRecipe } from './pages/RecipePages'
+import { CreateMealPlan, MealPlanFeed, SingleMealPlan } from './pages/MealPlanPages'
+import ProtectedRoute from './components/ProtectedRoute'
+import ApiClient from './assets/apiClient'
+
 
 function App() {
+  // shows whether the user is or isn't logged in
+  const [isLoggedIn, isLoggedInHandler] = useState(false);
+  let apiClient = new ApiClient();
+
+
+  let logInFunc = (userToken) => {
+    isLoggedInHandler(true)
+    window.localStorage.setItem("UserToken", userToken);
+  }
+  
+  let logOutFunc = () => {
+    window.localStorage.removeItem("UserToken");
+    isLoggedInHandler(false)
+  }
+
+  // uncomment line 31 to logout current use
+  window.localStorage.removeItem("UserToken")
   return (
-    <div className="App">
+    <div className="App py-5">
       <Router>
```

Comment:
We need to add an unprotected route for the welcome / intro page

Reply:
unprotected route for login/signup page is on line 35 - although this route does redirect to dashboard if user is logged in so they can't login more than once

Reply:
the welcome page is something i have forgetton to do to honest  - so i can look at that as part of a seperate issue